README.md

```diff
@@ -145,6 +152,117 @@ readData(dataInput) {
 }
 ```
 
+# Using Node.js Client with Hazelcast IMDG
+
+## 1. Node.js Client API Overview
+
+The first step is configuration. You can configure the Node.js client declaratively or programmatically. We will use the programmatic approach throughout this chapter. Please refer to the [Node.js Client Declarative Configuration section](#declarative-configuration) for details.
+
+```javascript
+var clientConfig = new Config.ClientConfig();
+clientConfig.groupConfig.name = 'dev';
+clientConfig.networkConfig.addresses.push('10.90.0.1', '10.90.0.2:5702');
```

Comment:
i think one address would be enough in here. One being with port and the other being without port is also confusing.

Reply:
I want to be consistent with Java documentatiton. See http://docs.hazelcast.org/docs/3.10.4/manual/html-single/index.html#client-api